File: Calculator.py
def add(x, y):
    return x + y

def subtract(x, y):
    return x - y

def multiply(x, y):
    return x * y

def divide(x, y):
    if y != 0:
        return x / y
    else:
        return "Erro: Divisão por zero"

while True:
    print("Escolha a operação:")
    print("1. Adição")
    print("2. Subtração")
    print("3. Multiplicação")
    print("4. Divisão")
    print("5. Sair")

    choice = input("Digite a opção (1/2/3/4/5): ")

    if choice == '5':
        print("Saindo...")
        break

    num1 = float(input("Digite o primeiro número: "))
    num2 = float(input("Digite o segundo número: "))

    if choice == '1':
        print("Resultado:", add(num1, num2))
    elif choice == '2':
        print("Resultado:", subtract(num1, num2))
    elif choice == '3':
        print("Resultado:", multiply(num1, num2))
    elif choice == '4':
        print("Resultado:", divide(num1, num2))
    else:
        print("Opção inválida")

File: calculator.js
<!DOCTYPE html>
<html>
<head>
    <title>Calculadora em JavaScript</title>
    <script>
        function add(x, y) {
            return x + y;
        }

        function subtract(x, y) {
            return x - y;
        }

        function multiply(x, y) {
            return x * y;
        }

        function divide(x, y) {
            if (y !== 0) {
                return x / y;
            } else {
                return "Erro: Divisão por zero";
            }
        }

        function calculate() {
            var choice = document.getElementById("operation").value;
            var num1 = parseFloat(document.getElementById("num1").value);
            var num2 = parseFloat(document.getElementById("num2").value);
            var result;

            if (choice === '1') {
                result = add(num1, num2);
            } else if (choice === '2') {
                result = subtract(num1, num2);
            } else if (choice === '3') {
                result = multiply(num1, num2);
            } else if (choice === '4') {
                result = divide(num1, num2);
            } else {
                result = "Opção inválida";
            }

            document.getElementById("result").innerHTML = "Resultado: " + result;
        }
    </script>
</head>
<body>
    <h2>Calculadora em JavaScript</h2>
    <label for="operation">Escolha a operação:</label>
    <select id="operation">
        <option value="1">Adição</option>
        <option value="2">Subtração</option>
        <option value="3">Multiplicação</option>
        <option value="4">Divisão</option>
    </select>
    <br>
    <label for="num1">Digite o primeiro número:</label>
    <input type="number" id="num1">
    <br>
    <label for="num2">Digite o segundo número:</label>
    <input type="number" id="num2">
    <br>
    <button onclick="calculate()">Calcular</button>
    <p id="result"></p>
</body>
</html>
